backend/agents/persona_agent.py

```
@@ -120,40 +128,353 @@ def _create_persona_tools(self) -> List[BaseTool]:
+                        knowledge_parts = []
+                        for doc in docs:
+                            knowledge_parts.append(f"- {doc.page_content}")
+                        return f"Relevant knowledge for {self.persona.name}:\n" + "\n".join(knowledge_parts)
+                    else:
+                        # Store the persona background for future reference
+                        self.vectorstore.add_texts(
+                            [f"{self.persona.name} background: {self.persona.background}"],
+                            metadatas=[{
+                                "persona_id": str(self.persona.id),
+                                "context_type": "knowledge",
+                                "timestamp": str(datetime.now())
```

Ruff (0.14.1)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)